CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
 # How to contribute
 
-If you encounter any bugs with the library please file an issue in the [Issues](https://github.com/Azure/azure-sdk-for-java/issues) section of the project.
+Thank you for your interest in contributing to Azure SDK for Java.
 
-If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.com/guidelines.html).
\ No newline at end of file
+# Ways to contribute
```

Comment:
Nice work.

Consider ripping off this https://github.com/dotnet/corefx/blob/master/CONTRIBUTING.md and refactoring the doc into 2 sections -  contributing information (coding styles, source structure, making pull requests, working with the team); and a separate Developer guide (how to enlist, build, run tests)

Reply:
to be clear, I'm OK merging this to get started and we can have subsequent PRs to add more to the document.  This is certainly much better than our current contributing.md 😄

Reply:
ok. I have updated it to have 2 main sections. Let me know what you think.

I am pointing to the docs we already have in the Wiki.  Probably we would prefer to have all docs in the repo itself instead of the wiki (or generate some static pages with Jekyll or something into github-pages instead of the simple github wiki)

Let me know what you think.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/5370#discussion_r324388946